intro-to-rails-2-relationships/ar-relationships.md

```diff
@@ -194,6 +197,165 @@ And, for the table-oriented among you:
 |       `author.books.build(book_hash)`       |     Book object     |         No         |   **Does** set `book.author_id`   |
 |      `author.books.create(book_hash)`       |     Book object     |        Yes         |   **Does** set `book.author_id`   |
 
-### Try it out!
+### Step by Step Guide
+
+Now let's try it out for ourselves. We're going to run through the steps to update our book app to have this relationship using our step-by-step buide.
+
+[AR Relationships Guide](active-record-relationships-exercise.resource.md).
```

Comment:
Has this already been moved over to Learn? Does a relative path like this work ok in Learn?

Reply:
Yep, I moved it into learn and made it a `.resource` file.